tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -62,12 +74,12 @@ function configure_cri_runtime() {
 	crio)
 		configure_crio
 		;;
-	containerd | k3s | k3s-agent)
+	containerd | k3s*)
 		configure_containerd
 		;;
 	esac
-	systemctl daemon-reload
-	systemctl restart "$1"
+	
+	restart_service $1
```

Comment:
```suggestion
	restart_service "$1"
```

Sorry, I'm paranoid about not quoting those.

Reply:
you actually have good point here